node/overseer/src/subsystems.rs

```diff
@@ -113,10 +113,18 @@ pub struct AllSubsystems<
 	pub approval_voting: ApV,
 	/// A Connection Request Issuer subsystem.
 	pub gossip_support: GS,
+	/// A Dispute Coordinator subsystem.
+	pub dispute_coordinator: DC,
+	/// A Dispute Participation subsystem.
+	pub dispute_participation: DP,
+	/// A Dispute Distribution subsystem.
+	pub dispute_distribution: DD,
+	/// A Chain Selection subsystem.
+	pub chain_selection: CS,
```

Comment:
Does this automatically generate communication?

Reply:
Not here, this is a barely used abstraction for `Overseer::new` which will be removed soon™ ( tracked as chore in https://github.com/paritytech/polkadot/issues/3427 )